### DISCUSSION TITLE
`participantId` in catalog

### DISCUSSION THREAD

**Discussion by arnoweiss** [eclipse-edc/Connector#4311] [General]
I've observed that the `/v2/catalog/request` API returns a payload with properties called `participantId` and `dspace:participantId` holding the same data. 

```json
{
    "@id": "8b935442-8045-43d9-b7db-3a464c57ae61",
    "@type": "dcat:Catalog",
    "dspace:participantId": "BPNL0000000ABCDE",
    "dcat:dataset": [ ],
    "dcat:service": {
        "@id": "f40e7a04-4638-4240-bcb8-92dde5e67a9d",
        "@type": "dcat:DataService",
        "dcat:endpointDescription": "dspace:connector",
        "dcat:endpointUrl": "https://mycontrol.plane/api/v1/dsp",
        "dct:terms": "dspace:connector",
        "dct:endpointUrl": "https://mycontrol.plane/api/v1/dsp"
    },
    "participantId": "BPNL0000000ABCDE",
    "@context": {
        "@vocab": "https://w3id.org/edc/v0.0.1/ns/",
        "edc": "https://w3id.org/edc/v0.0.1/ns/",
        "tx": "https://w3id.org/tractusx/v0.0.1/ns/",
        "tx-auth": "https://w3id.org/tractusx/auth/",
        "cx-policy": "https://w3id.org/catenax/policy/",
        "dcat": "http://www.w3.org/ns/dcat#",
        "dct": "http://purl.org/dc/terms/",
        "odrl": "http://www.w3.org/ns/odrl/2/",
        "dspace": "https://w3id.org/dspace/v0.8/"
    }
}
```

In both json and (considering the `@context`) json-ld logic, these are different properties. The DSP-schema exclusively mentions `dspace:participantId`. However, it seems plausible that they are duplicate. Is this current behavior intentional - for instance for compatibility reasons? If it isn't, I'll convert this to an issue. 

This was observed with v0.7.0 (imported by 0.7.2 tractusx-edc).

**Comment by ndr-brt** [comment]
this was already discussed (e.g. https://github.com/eclipse-edc/Connector/issues/4231), the "duplicated" one was there before the last DSP changes, and it will stay there until its deprecation will expire
